tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js

```diff
@@ -0,0 +1,14 @@
+export function sumFibs(num) {
+  let first = 1;
+  let second = 1;
+  let sumOfFibs = 1;
+  while (num >= second) {
+    if (second % 2 === 1) {
+      sumOfFibs += second;
+    }
+    let temp = second;
+    second += first;
+    first = temp;
```

Comment:
This `temp` variable seems to be redundant

```suggestion
    second += first;
    first = second - first;
```